airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -172,7 +180,15 @@ def __normalize_identifier_name(
             else:
                 result = result.replace("`", "_")
             result = result.replace("'", "\\'")
-            result = f"adapter.quote('{result}')"
+            if self.destination_type == DestinationType.ORACLE:
+                # Oracle dbt lib doesn't implemented adapter quote yet.
+                result = self.__normalize_identifier_case(result, is_quoted=True)
+                result = f"quote('{result}')"
+                if not in_jinja:
```

Comment:
Do you need this for oracle?

Reply:
I removed because there is a if with the same function bellow.